Gems/Atom/RPI/Code/Source/RPI.Builders/Material/MaterialBuilder.cpp

```diff
@@ -93,8 +94,12 @@ namespace AZ
                     {
                         AssetBuilderSDK::JobDependency jobDependency;
                         jobDependency.m_jobKey = jobKey;
-                        jobDependency.m_type = AssetBuilderSDK::JobDependencyType::Order;
                         jobDependency.m_sourceFile.m_sourceFileDependencyPath = file;
+
+                        bool includeMaterialPropertyNames = true;
+                        RPI::MaterialConverterBus::BroadcastResult(includeMaterialPropertyNames, &RPI::MaterialConverterBus::Events::ShouldIncludeMaterialPropertyNames);
```

Comment:
This setting should not be used here. MaterialConverter is only part of the SceneAPI build pipeline, for converting FBX (etc) materials to Atom materials. Notice the settings appear under "/O3DE/SceneAPI/MaterialConverter", they are specific to SceneAPI.

I think we can either keep the default of includePropertyNames=true and don't make it configurable for now, or add a new setreg section for MaterialBuilder settings (in that case I suggest "/O3DE/Atom/RPI/MaterialBuilder/IncludeMaterialPropertyNames").

Reply:
We should probably have a comment to explain why we are using OrderOnce.

Reply:
Fixed, added registry setting.